instrumentation/activej-http-6.0/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/activejhttp/ActivejRoutingServletTest.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.activejhttp;
+
+import static io.activej.http.HttpMethod.GET;
+import static io.opentelemetry.sdk.testing.assertj.OpenTelemetryAssertions.equalTo;
+import static io.opentelemetry.semconv.HttpAttributes.HTTP_REQUEST_METHOD;
+import static io.opentelemetry.semconv.HttpAttributes.HTTP_RESPONSE_STATUS_CODE;
+import static io.opentelemetry.semconv.UrlAttributes.URL_PATH;
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import io.activej.eventloop.Eventloop;
+import io.activej.http.AsyncServlet;
+import io.activej.http.HttpError;
+import io.activej.http.HttpRequest;
+import io.activej.http.HttpResponse;
+import io.activej.http.RoutingServlet;
+import io.activej.http.UrlParser;
+import io.activej.promise.Promise;
+import io.activej.reactor.Reactor;
+import io.opentelemetry.api.trace.SpanKind;
+import io.opentelemetry.instrumentation.testing.junit.AgentInstrumentationExtension;
+import io.opentelemetry.instrumentation.testing.junit.InstrumentationExtension;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+class ActivejRoutingServletTest {
+
+  @RegisterExtension static final EventloopExtension eventloopExtension = new EventloopExtension();
+
+  @RegisterExtension
+  static final InstrumentationExtension testing = AgentInstrumentationExtension.create();
+
+  private static final Eventloop eventloop = Reactor.getCurrentReactor();
+
+  @Test
+  void testRequestSuccessFlow() throws Exception {
+    AsyncServlet asyncServlet =
+        request -> HttpResponse.ofCode(200).withBody("".getBytes(UTF_8)).toPromise();
+
+    RoutingServlet routingServlet =
+        RoutingServlet.builder(eventloop).with(GET, "/success", asyncServlet).build();
+
+    String url = "http://some-test.com/success";
+    HttpRequest httpRequest = HttpRequest.get(url).build();
+    check(routingServlet.serve(httpRequest), "", 200);
+
+    UrlParser urlParser = UrlParser.of(url);
+
+    testing.waitAndAssertTraces(
+        trace ->
+            trace.hasSpansSatisfyingExactly(
+                span ->
+                    span.hasName("GET")
+                        .hasNoParent()
+                        .hasKind(SpanKind.SERVER)
+                        .hasAttributesSatisfying(
```

Comment:
Could you do the more strict check?
```suggestion
                        .hasAttributesSatisfyingExactly(
```